### DISCUSSION TITLE
Runner not picking job

### DISCUSSION THREAD

**Discussion by mkapra** [community/community#36320] [Actions]
I have a matrix of OSs building my 'software'. The problem is that for each OS a runner
picks up the job. Only the fedora job gets not picked up. Any ideas on that?

Latest action with that problem: https://github.com/mkapra/vpnctrl/actions/runs/3243195457

**Comment by vitalyk-multinarity** [comment]
AFAIK, GitHub doesn't provide Fedora distro on their runners- https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.
So if you have to run your workflow on Fedora you should launch self-hoster runner.

**Comment by mkapra** [comment]
Ah okay thanks! I did not thought about that

**Comment by ossanna16** [comment]
Hey there @mkapra and thank you for asking a good question! I'm glad you found our community 🙂
